src/Selenium2Driver.php

```diff
@@ -325,7 +325,22 @@ private function applyTimeouts()
             throw new DriverException('Error setting timeout: ' . $e->getMessage(), 0, $e);
         }
     }
+      /**
+     * {@inheritdoc}
```

Comment:
The DocBlock can't be inherited because there are no such method called `acceptAlert` in parent class.